airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/core/context/env/ConnectorConfigurationPropertySource.kt

```diff
@@ -127,6 +127,7 @@ class ConnectorConfigurationPropertySource(commandLine: CommandLine) :
             }
         }
 
+        @Suppress("UNCHECKED_CAST")
```

Comment:
why is this suddenly necessary when the previous PR passed all the checks?

Reply:
I'm not 100% sure.  I think that this is another example of something being enforced once we bumped the Kotlin version.